salt/modules/ini_manage.py

```diff
@@ -318,17 +318,18 @@ def gen_ini(self):
         yield '{0}[{1}]{0}'.format(os.linesep, self.name)
         sections_dict = OrderedDict()
         for name, value in six.iteritems(self):
+            # Handle Comment Lines
             if com_regx.match(name):
                 yield '{0}{1}'.format(value, os.linesep)
+            # Handle Sections
             elif isinstance(value, _Section):
                 sections_dict.update({name: value})
+            # Key / Value pairs
+            # Adds spaces between the separator
             else:
                 yield '{0}{1}{2}{3}'.format(
                     name,
-                    (
-                        ' {0} '.format(self.sep) if self.sep != ' '
-                        else self.sep
-                    ),
+                    ' {0} '.format(self.sep) if self.sep != ' ' else self.sep,
```

Comment:
I'm not sure that I follow the reasoning for this change. Could you please clarify?

Reply:
It just didn't need to be wrapped in a parenthesis...